doc/extended-configuration.md

```diff
@@ -140,6 +140,18 @@ You may want to provide a cron expression to configure how frequently the ip-rec
 
 Look for the following parameter `"reconciler_cron_expression"` located in `script/install-cni.sh` and change to your desired schedule.
 
+alternatively you can speficy the `WHEREABOUTS_RECONCILER_CRON` environment variable in your daemonset definition file to provide the cron expresion.
+
+```
+       env:
+        - name: WHEREABOUTS_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
```

Comment:
I would omit this unrelated variable from the snippet. It is unrelated.

Reply:
done